translations/ja.json

```diff
@@ -598,46 +598,46 @@
 	"Target {{.path}} can not be empty": "",
 	"Test docs have been saved at - {{.path}}": "",
 	"The \"{{.driver_name}}\" driver requires root privileges. Please run minikube using 'sudo minikube --vm-driver={{.driver_name}}": "「{{.driver_name}}」ドライバにはルート権限が必要です。「sudo minikube --vm-driver={{.driver_name}}」を使用して minikube を実行してください",
-	"The \"{{.driver_name}}\" driver should not be used with root privileges.": "",
-	"The \"{{.name}}\" cluster has been deleted.": "「{{.name}}」クラスタが削除されました",
-	"The \"{{.name}}\" cluster has been deleted.__1": "「{{.name}}」クラスタが削除されました",
-	"The 'none' driver is designed for experts who need to integrate with an existing VM": "",
-	"The 'none' driver provides limited isolation and may reduce system security and reliability.": "ドライバに「none」を指定すると、分離が制限され、システムのセキュリティと信頼性が低下する可能性があります",
```

Comment:
「」is forbidden in https://github.com/kubernetes/dashboard/tree/master/i18n/ja#%E5%92%8C%E8%A8%B3%E4%B8%80%E8%88%AC ?
Sorry, I can't find the rule in this.

Reply:
I've fixed it. Please check the latest commit.